pkg/output/json.go

```diff
@@ -52,12 +59,41 @@ func newResourceJSON(r *schema.Resource) resourceJSON {
 	}
 }
 
-func ToJSON(resources []*schema.Resource) ([]byte, error) {
+func showSkippedResourcesJSON(resources []*schema.Resource, showDetails bool) []string {
+	unSupportedTypeCount, _, unSupportedCount, _ := terraform.CountSkippedResources(resources)
+	if unSupportedCount == 0 {
+		return nil
+	}
+	message := fmt.Sprintf("\n%d out of %d resources couldn't be estimated as Infracost doesn't support them yet (https://www.infracost.io/docs/supported_resources)", unSupportedCount, len(resources))
+	if showDetails {
+		message += ".\n"
+	} else {
+		message += ", re-run with --show-skipped to see the list.\n"
+	}
+	message += "We're continually adding new resources, please create an issue if you'd like us to prioritize your list.\n"
+	if showDetails {
+		for rType, count := range unSupportedTypeCount {
+			message += fmt.Sprintf("%d x %s\n", count, rType)
+		}
+	}
+	return []string{message}
```

Comment:
Should we just return a string instead of []string?

Reply:
Right, will fix it.

Reply:
The whole function was removed.